docs/en/operation/Journal.md

```diff
@@ -93,3 +94,49 @@ If the restore succeeds, you should see a log message along the lines of
 INFO AlluxioMasterProcess - Restored 57 entries from backup
 ```
 in the primary master logs.
+
+### Changing Masters
+
+If the journal is stored in a distributed system like HDFS, changing masters is easy.
+As long as the new master sets `alluxio.master.journal.folder` the same as the old
+master, it will start up in the same state that the old master left off.
+
+If the journal is stored on the local filesystem, the journal folder needs to be
+copied to the new master.
+
+## Advanced
+
+### Journal Checkpoints
```

Comment:
Users may not really understand checkpoints in the beginning, but they really need to solve is to "bound the restart time" or "reduce the storage for log files", can we re-state the same content based on users' motivation?

Reply:
Great suggestion, renamed this section to "Managing the journal size"